src/_lib/sass/src/_email-input.scss

```diff
@@ -57,12 +56,33 @@ nw-email-input {
             height: 0;
             padding: 0;
             margin: 0;
-        }
+        }      
     }
     .pill-hidden {
         visibility: hidden;
         height: 0 !important;
         margin: 0;
         font-size: 16px !important;
     }
+
+    .persistent-placeholder{
+        input {
+            width: 100%;
+            height: 100%;  
+            background-color: transparent;
+            padding-bottom: 6px;
+            &:focus {
+                margin: 0;
+                margin-left: -2px;
+            }
+            &:not(:focus) {
```

Comment:
You probably don't need this selector here you can just include the styles inside the input above directly no ?